js/player.js

```diff
@@ -16,6 +16,7 @@ function($, playlist, ytIframe){
 
         $('#play_prev').on('click', playPrev);
         $('#play_next').on('click', playNext);
+        $('#clear_playlist').on('click', clearPlaylist);
```

Comment:
When you move the `clearPlaylist` function to playlist.js, you can access it using `playlist.clearPlaylist`